cmds/core/netcat/netcat.go

```diff
@@ -102,7 +106,7 @@ func evalParams(args []string, f flags) (*netcat.Config, error) {
 
 			port, err := strconv.ParseUint(ports[0], 10, 64)
 			if err != nil {
-				return nil, fmt.Errorf("invalid port: %w", err)
+				return nil, fmt.Errorf("invalid port: %v", ports[0])
```

Comment:
you could consider wrapping os.ErrInvalid here so that writing a test is easier at some point.

Reply:
I added a commit which wrapps os.ErrInvalid in all the parsing errors.